apps/mull-ui/src/app/pages/create-event/create-event.scss

```diff
@@ -0,0 +1,80 @@
+@import 'variables';
+
+.create-event {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 95%;
+  margin: auto;
+  .custom-text-input-container {
+    margin-bottom: 1.875rem;
+  }
+
+  .custom-time-picker-container {
+    margin: 0 0 1.25rem;
+  }
+}
+
+.event-image-upload {
+  display: none;
+}
+
+.event-input-border {
+  border-radius: 8px;
+  border: 1.5px $primary-color solid;
+  &.error {
+    border-color: #ff0000;
+  }
+}
+
+.custom-file-upload {
+  display: block;
+  padding: 1.8em 5em;
+  cursor: pointer;
+
+  margin-bottom: 20px;
+
+  &:hover {
+    background: #0ba024;
+  }
+}
+
+.create-event-text {
+  color: $primary-color;
+  font-size: 16px;
+  margin-bottom: 0.5rem;
+}
+
+.create-event-label {
+  color: black;
+  align-items: flex-start;
+  font-weight: 600;
+}
+
+.create-event-input {
+  display: block;
+  box-sizing: border-box;
+  outline: none;
+  padding-left: 1em;
+  width: 100%;
+  height: 2.8em;
+}
+
+.create-event-button {
+  color: white;
+  font-weight: bold;
+  padding: 15px 55px;
+  background: $primary-color;
+  border: 1px solid $primary-color;
+  border-radius: 9px;
+  outline: none;
+  margin-top: 20px;
+  margin-bottom: 15px;
+}
+
+@media (min-width: $breakpoint-mobile) {
+  .create-event {
+    padding: 1rem 0 0;
+  }
```

Comment:
Text is too small on desktop
Mockup:
![image](https://user-images.githubusercontent.com/23544999/98428389-3fdf7580-206f-11eb-9ab9-37f998dc3301.png)
Implementation:
![image](https://user-images.githubusercontent.com/23544999/98428408-5c7bad80-206f-11eb-9f10-8ec9105c1662.png)

```suggestion
    .create-event {
    padding: 1rem 0 0;
  }
  .create-event-text {
    font-size: 1.5rem;
  }
```